experiments/snippets/realTimeOppDashboard/README.md

```diff
@@ -0,0 +1,7 @@
+# Real Time Opportunity Dashboard
+
+Skuid pages generally only query the database on user request.  The user reloads the page, or takes some action on the page to trigger a query.  But customers often ask whether Skuid can create a page where new data is "pushed".  
+
+This page extends Skuid with a simple snippet that polls a "updates indicator" model periodically.  If this model returns data - the user can be warned that there are updates and can update the page. 
+
+It is not wise to simply continuously poll the primary model, as that would freeze the components bound to that model whenever the queries are occuring,  and that cause problems if there were unsaved chagnes. 
```

Comment:
Minor style tweaks, a lot of adverbs here. Also spelling, verb tense, and whitespace.
```suggestion
It is not wise to just continuously poll the primary model, as that would freeze the components bound to that model whenever the queries occur. These frozen components would then cause problems for any unsaved changes. 
```

Reply:
@Rnhatch

Reply:
I'm going to rewrite sentence cuz I'm not happy with either option.